dashboard/app/helpers/levels_helper.rb

```diff
@@ -443,6 +445,7 @@ def render_app_dependencies
   #  3. The disable_google_blockly DCDO flag, which contains an array of strings corresponding to model class names.
   #     This option will override #2 as an "emergency switch" to go back to CDO Blockly.
   def use_google_blockly
+    return true if @level.uses_lab2?
```

Comment:
Should we default to the level setting here? This won't always be true when we have non-Blockly labs right? Though FWIW since we're not depending on `apps_dependencies` do we even check this?

Reply:
It seems we're no longer using this, so I've just removed the line.